README.md

```diff
@@ -57,15 +57,15 @@ $ docker run -t -i -p 8080:8080 cockroachdb/cockroach shell
 root@82cb657cdc42:/cockroach#
 ```
 
-Now we're in an environment that has everything set up, and we start by first initializing the cluster and then firing up a node:
+Now we're in an environment that has everything set up, and we start by first initializing the cluster and then firing up the node:
 
 ```bash
 $ DIR=$(mktemp -d /tmp/dbXXX)
-$ ./cockroach init $DIR"
-$ ./cockroach start -stores ssd="$DIR" &> node.log &
+$ ./cockroach init $DIR
+$ ./cockroach start -stores ssd="$DIR" -gossip :8080 &
```

Comment:
I would be explicit here and say `-gossip localhost:8080`. Using an implied hostname sounds like this controls server-side behavior when it is actually the gossip client.